src/components/blocks/InformationBlock/InformationBlock.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { render, screen } from '@testing-library/react';
+import InformationBlock from './InformationBlock';
+
+describe('InformationBlock', () => {
+  it('renders with given text', () => {
```

Comment:
This test is not enough to cover all functionality

Test cases I see:
1. Renders with variant=main
2. Renders with variant=regular
3. Renders data in row/column (assign `data-testid` to div with dataInRow class and look for it by `screen.getByTestId` method)
4. Renders with given classname

Reply:
@vadim-kirpun i need some clarifications on point 3. InformationBlock doesn't accept children, and `dataInRow` is a boolean. How can I assign `data-testid`?

Reply:
@winston0410 inside of InformationBlock like this
![image](https://user-images.githubusercontent.com/86728658/152145672-33de5e85-bbca-4d98-8d43-6acc03a27c7c.png)

Reply:
Ok. Will this be stripped off in production?

Reply:
@winston0410 yes it should be removed automatically

Reply:
I see